python/paddle/fluid/layers/nn.py

```diff
@@ -7580,3 +7582,97 @@ def hash(input, hash_size, num_hash=1, name=None):
         attrs={'num_hash': num_hash,
                'mod_by': hash_size})
     return out
+
+
+def log_loss(input, label, epsilon=1e-4, name=None):
+    """
+    **Negative Log Loss Layer**
+
+    This layer accepts input predictions and target label and returns the
+    negative log loss.
+
+    .. math::
+
+        Out = -label * log(input + epsilon)
+              - (1 - label) * log(1 - input + epsilon)
+
+    Args:
+        input (Variable|list):  a 2-D tensor with shape [N x 1], where N is the
+                                batch size. This input is a probability computed
+                                by the previous operator.
+        label (Variable|list):  the ground truth which is a 2-D tensor with
+                                shape [N x 1], where N is the batch size.
+        epsilon (float): epsilon
+
+    Returns:
+        A 2-D tensor with shape [N x 1], the negative log loss.
```

Comment:
describe return type at the begining like
```
Returns:
    type: description
```

see: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html